### ISSUE TITLE
Exception handling in HW1-1

### ISSUE THREAD

**Issue by dooly9931** [swsnu/swppfall2021#8]
Hi,
As I understood, for `safe_internet_fetch` and `BabyFetchException`, we are told to check `urllib.error.URLError` for invalid url or unstable internet connection. But since there are so many ways to make url invalid, `urllib.error.HTTPError` or `http.client.InvalidURL` are raised instead of `urllib.error.URLError` for some cases. Is it okay to check only `urllib.error.URLError` when we are raising `BabyFetchException`?
Thank you in advance.

**Comment by ttoru96** [comment]
Yes, it is good enough if you've checked `urllib.error.URLError`.

**Comment by dooly9931** [comment]
Thanks!
